server/apps/fieldsight/loader.py

```diff
@@ -156,13 +161,14 @@ def load_geosite(self, datum):
         defaults['longitude'] = get_attr(datum, 'longitude') or \
             datum['location'][0]
 
-        defaults['district'], _ = District.objects.get_or_create(
-            name=get_attr(datum, 'District')
-        )
-        defaults['gaupalika'], _ = Gaupalika.objects.get_or_create(
-            name=get_attr(datum, 'Gaupalika'),
-            defaults={'district': defaults['district']}
-        )
+        defaults['district'] = District.objects.filter(
+            name__trigram_similar=get_attr(datum, 'District')
+        ).first()
```

Comment:
We should have a threshold for trigram similarity

Reply:
Created an issue for this at https://github.com/toggle-corp/pin-dashboard/issues/33

Reply:
Done